PhotoFeed/app/src/main/java/com/example/yhuang/photofeed/HttpRequest/ImageLoadTask.java

```diff
@@ -0,0 +1,46 @@
+package com.example.yhuang.photofeed.HttpRequest;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.os.AsyncTask;
+import android.widget.ImageView;
+
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+/**
+ * Created by yhuang on 9/26/2015.
+ */
+public class ImageLoadTask extends AsyncTask<Void, Void, Bitmap> {
+    private String url;
+    private ImageView imageView;
+
+    public ImageLoadTask(String url, ImageView imageView) {
+        this.url = url;
+        this.imageView = imageView;
+    }
+
+    @Override
+    protected Bitmap doInBackground(Void... params) {
+        try {
+            URL urlConnection = new URL(url);
+            HttpURLConnection connection = (HttpURLConnection) urlConnection
+                    .openConnection();
+            connection.setDoInput(true);
+            connection.connect();
+            InputStream input = connection.getInputStream();
+            Bitmap myBitmap = BitmapFactory.decodeStream(input);
+            return myBitmap;
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        return null;
```

Comment:
You could move this null inside the try for more clarity and the code will function the same way.